gremlinapi/scenarios.py

```diff
@@ -267,6 +267,8 @@ def update_scenario_result_notes(
             f"/scenarios/{guid}/runs/{run_number}/resultNotes", **kwargs
         )
         payload: dict = cls._payload(**{"headers": https_client.header(), "body": data})
+        # Content-Type must be text/plain for this endpoint
+        payload['headers']['Content-Type'] = 'text/plain'
```

Comment:
I'm reviewing the list of tests/ now; but do you know if any of these need updated to align with this change?

Reply:
I can jump back into it and look. I added this a few weeks ago and we just now got the pypi process fixed.

Reply:
In discussion, sounds like there's larger work with test maintenance.  Not going to force a slight change here in the face of that larger check.